[2024-06-18T11:05:41.217+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T11:05:41.233+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag.second_task scheduled__2023-10-07T00:00:00+00:00 [queued]>
[2024-06-18T11:05:41.238+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag.second_task scheduled__2023-10-07T00:00:00+00:00 [queued]>
[2024-06-18T11:05:41.239+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-18T11:05:41.249+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): second_task> on 2023-10-07 00:00:00+00:00
[2024-06-18T11:05:41.253+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4357) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-18T11:05:41.254+0000] {standard_task_runner.py:63} INFO - Started process 4358 to run task
[2024-06-18T11:05:41.254+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'second_task', 'scheduled__2023-10-07T00:00:00+00:00', '--job-id', '689', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpbwfqb98q']
[2024-06-18T11:05:41.255+0000] {standard_task_runner.py:91} INFO - Job 689: Subtask second_task
[2024-06-18T11:05:41.285+0000] {task_command.py:426} INFO - Running <TaskInstance: test_dag.second_task scheduled__2023-10-07T00:00:00+00:00 [running]> on host 195daacf7793
[2024-06-18T11:05:41.345+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_dag' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2023-10-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-07T00:00:00+00:00'
[2024-06-18T11:05:41.345+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T11:05:41.354+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-18T11:05:41.354+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2024-06-18T11:05:41.360+0000] {subprocess.py:86} INFO - Output:
[2024-06-18T11:05:41.361+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2024-06-18T11:05:41.362+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-18T11:05:41.362+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T11:05:41.379+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=second_task, run_id=scheduled__2023-10-07T00:00:00+00:00, execution_date=20231007T000000, start_date=20240618T110541, end_date=20240618T110541
[2024-06-18T11:05:41.428+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-18T11:05:41.439+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-18T11:05:41.440+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
